crates/gpio/src/vhu_gpio.rs

```diff
@@ -505,7 +505,7 @@ mod tests {
 
     use super::Error;
     use super::*;
-    use crate::gpio::tests::DummyDevice;
+    use crate::gpio::DummyDevice;
```

Comment:
Since we have `use crate::gpio::*;` maybe we can remove this line

Reply:
No longer relevant as we moved it to:
```
    use crate::mock_gpio::MockGpioDevice;
```